### PR TITLE
docs(README.md): fix typos, add trailing newline

### PR DESCRIPTION
**Pull request by lloydchang**
Signed-off-by: lloydchang <lloydchang@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,21 +253,21 @@ CNCF doesn't set a set amount of budget for each project and will work with you
...
 
-Please note that available computing resources are limited so we may ask you to reduce your usage when there is high demand for the available credits. Specifically, please consider shutting down the unused computing resources, use automation to terminate the bare metal/virtual machines if they are not intended to be used 24/7, use spot instances if applicable etc. Please estimate your budget to use no more that $3000/month in AWS credits. If you expect higher resources usage on the regular basis, please consider using the [CNCF Cluster](https://github.com/CNCF/cluster) instead.
+Please note that available computing resources are limited so we may ask you to reduce your usage when there is high demand for the available credits. Specifically, please consider shutting down the unused computing resources, use automation to terminate the bare metal/virtual machines if they are not intended to be used 24/7, use spot instances if applicable etc. Please estimate your budget to use no more than $3000/month in AWS credits. If you expect higher resource usage on a regular basis, please consider using the [CNCF Cluster](https://github.com/CNCF/cluster) instead.
```
Comment (lloydchang) [inline README.md:270]:
```suggestion
Please note that available computing resources are limited so we may ask you to reduce your usage when there is high demand for the available credits. Specifically, please consider shutting down the unused computing resources, use automation to terminate the bare metal/virtual machines if they are not intended to be used 24/7, use spot instances if applicable etc. Please estimate your budget to use no more than $3000/month USD in AWS credits. If you expect higher resource usage on a regular basis, please consider using the [CNCF Cluster](https://github.com/CNCF/cluster) instead.
```

File: README.md [hunk 2]
```diff
@@ -253,21 +253,21 @@ CNCF doesn't set a set amount of budget for each project and will work with you
...
 
 ### How can I use the computing infrastructure by CNCF?
 
-CNCF prefers projects evaluate using our [Community Cluster](https://github.com/cncf/cluster) first. We have partnered with the various providers that offer discounted or free services for the CNCF projects. For example, the CNCF projects may use the credits, [offered by Amazon Web Services for CNCF](https://www.cncf.io/announcement/2019/11/19/cloud-native-computing-foundation-receives-200000-in-credits-from-amazon-web-services-aws/) for the upstream testing, CI/CD and other purposes. See the [Tools](#tools) sections on this page for more details.
+CNCF prefers projects evaluated using our [Community Cluster](https://github.com/cncf/cluster) first. We have partnered with the various providers that offer discounted or free services for the CNCF projects. For example, the CNCF projects may use the credits, [offered by Amazon Web Services for CNCF](https://www.cncf.io/announcement/2019/11/19/cloud-native-computing-foundation-receives-200000-in-credits-from-amazon-web-services-aws/) for the upstream testing, CI/CD and other purposes. See the [Tools](#tools) sections on this page for more details.
```
Comment (amye) [inline README.md:264]: Not necessary

**Review by amye — COMMENTED** [review]
One nit, but otherwise fine!

File: README.md (hunk 2 above)
Comment (lloydchang) [inline README.md:264]:
```suggestion
CNCF prefers projects evaluated using our [Community Cluster](https://github.com/cncf/cluster) first. We have partnered with the various providers that offer discounted or free services for the CNCF projects. For example, the CNCF projects may use the credits, [offered by Amazon Web Services for CNCF](https://www.cncf.io/announcement/2019/11/19/cloud-native-computing-foundation-receives-200000-in-credits-from-amazon-web-services-aws/) for the upstream testing, CI/CD and other purposes. See the [Tools](#tools) sections on this page for more details.
```

**Pull request merged**